### PR TITLE
All prettier

### PR DESCRIPTION
**Pull request by binnmti**
## Summary by CodeRabbit

- **Style**
	- Improved formatting and readability of the Edit component.
	- Standardized return statements and event handler formatting for consistency.
	- Enhanced clarity in JSX rendering logic.

- **Chores**
	- Added code formatting and linting steps to the CI workflow for improved development practices.
	- Integrated new dependencies for enhanced code formatting capabilities.
	- Modified the scheduled job to prevent execution during scheduled events, streamlining workflow.

### CODE REVIEW

File: one-pager-maker/package.json [hunk 1]
```diff
@@ -7,7 +7,9 @@
     "dev": "vite",
     "devEmu": "firebase emulators:exec --ui 'npm run dev'",
     "build": "tsc && vite build",
-    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "lint": "eslint src tests -c .eslintrc.cjs",
+    "format": "prettier --list-different \"src/**/*.{ts,tsx}\" \"tests/**/*.{ts,tsx}\"",
+    "fix": "bun run format && bun run lint",
```
Comment (Harineko0) [inline one-pager-maker/package.json:12]: 私のプロジェクトでは、lint コマンドをエラーの確認、fix コマンドを整形としています。
fix コマンドで自動整形されるのでおすすめです
```
    "lint": "eslint src tests -c eslint.config.mjs && prettier --check \"src/**/*.ts\" \"src/**/*.tsx\" \"tests/**/*.ts\"",
    "fix": "prettier --write \"src/**/*.ts\" \"src/**/*.tsx\" \"tests/**/*.ts\" && eslint --fix src tests",
```

**Review by Harineko0 — COMMENTED** [review]
レビューしました b

File: one-pager-maker/package.json (hunk 1 above)
Comment (binnmti) [reply one-pager-maker/package.json]: That's good. Thank you. I fixed.

**Review by Harineko0 — APPROVED** [review]
LGTM

**Pull request merged**